packages/neon-api/src/transaction/util.ts

```diff
@@ -0,0 +1,21 @@
+import { tx, wallet, u } from "@cityofzion/neon-core";
+
+export function checkMultisigAcc(
```

Comment:
this looks very specific to the multisig method. I would just put it into the signer class and private it. It doesnt look like it has a lot of use outside.

Reply:
OK